gulpfile.js/index.js

```diff
@@ -190,8 +190,8 @@ const plugins = series(languagePlugins, minifyPlugins);
 
 module.exports = {
 	watch: watchComponentsAndPlugins,
-	default: parallel(components, plugins, componentsJsonToJs, build),
 	premerge,
 	linkify,
-	changes
+	changes,
```

Comment:
What happened here?

Reply:
For some reason, after a gulp update, I gulp would not work for me without this modification.

Reply:
What? Changing the order of exported items shouldn't have any effect.
What error message did you get? 
Is this also the reason you down-graded Mocha?

Changes to `gulpfile.js/*`, `package.json`, and `package-lock.json` should not be part of a language PR.

Reply:
I'm honestly not sure what happened. I just replaced the gulp.js files with what's in master, and gulp ran fine. I'll commit those to revert.

Reply:
I'm not sure how to fix this travis build failure. Any suggestions?

Reply:
The reason the build failed is that you forgot to revert the `package-lock.json` and now they are out of sync which causes `npm ci` to fail.

> cipm can only install packages when your package.json and package-lock.json or npm-shrinkwrap.json are in sync. Please update your lock file with `npm install` before continuing.

Reply:
> I'm honestly not sure what happened. I just replaced the gulp.js files with what's in master, and gulp ran fine.

Good that it works now.

Reply:
> The reason the build failed is that you forgot to revert the `package-lock.json` and now they are out of sync which causes `npm ci` to fail.
> 
> > cipm can only install packages when your package.json and package-lock.json or npm-shrinkwrap.json are in sync. Please update your lock file with `npm install` before continuing.

Once I run npm install, I get the gulp error I got before:

```
[09:46:55] Using gulpfile ~\git\prism\gulpfile.js
[09:46:55] Task never defined: default
[09:46:55] To list available tasks, try running: gulp --tasks
````

Reply:
Cannot reproduce. I clone your repo and everything works on my side.
What version of Node are you using? (I'm using v10.4.0)

<details>

This is the Windows CMD log. Commands are the same on bash.

```
C:\Users\Michael\Desktop>git clone https://github.com/cedporter/prism.git
Cloning into 'prism'...
remote: Enumerating objects: 2, done.
remote: Counting objects: 100% (2/2), done.
remote: Compressing objects: 100% (2/2), done.
remote: Total 17571 (delta 0), reused 1 (delta 0), pack-reused 17569Receiving objects: 100% (17571/17571), 3.77 MiB | 3
Receiving objects: 100% (17571/17571), 5.85 MiB | 4.26 MiB/s, done.
Resolving deltas: 100% (11993/11993), done.

C:\Users\Michael\Desktop>cd prism

C:\Users\Michael\Desktop\prism>git status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean

C:\Users\Michael\Desktop\prism>git checkout sas-improvements
Switched to a new branch 'sas-improvements'
Branch 'sas-improvements' set up to track remote branch 'sas-improvements' from 'origin'.

C:\Users\Michael\Desktop\prism>git status
On branch sas-improvements
Your branch is up to date with 'origin/sas-improvements'.

nothing to commit, working tree clean

C:\Users\Michael\Desktop\prism>npm i
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.7 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.7: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

added 544 packages from 816 contributors and audited 6938 packages in 8.839s
found 95 high severity vulnerabilities
  run `npm audit fix` to fix them, or `npm audit` for details

C:\Users\Michael\Desktop\prism>npx gulp
[16:56:30] Using gulpfile ~\Desktop\prism\gulpfile.js
[16:56:30] Starting 'default'...
[16:56:30] Starting 'minifyComponents'...
[16:56:30] Starting 'componentsJsonToJs'...
[16:56:30] Starting 'build'...
[16:56:30] Starting 'languagePlugins'...
[16:56:30] Finished 'componentsJsonToJs' after 106 ms
[16:56:30] Finished 'languagePlugins' after 127 ms
[16:56:30] Starting 'minifyPlugins'...
[16:56:30] Finished 'build' after 203 ms
[16:56:31] Finished 'minifyPlugins' after 591 ms
[16:56:32] Finished 'minifyComponents' after 1.32 s
[16:56:32] Finished 'default' after 1.33 s
```

</details>

Reply:
Once I ran gulp with "npx gulp" it worked. Running simply "gulp" appears to have been the culprit.

Reply:
Just a guess but could it be that your global `gulp` and your local `npx gulp` are different versions?
Well, as long as it works for you, it's fine I guess.